gr-qtgui/lib/freq_sink_c_impl.cc

```diff
@@ -410,6 +419,22 @@ void freq_sink_c_impl::buildwindow()
     if (d_wintype != filter::firdes::WIN_NONE) {
         d_window = filter::firdes::window(d_wintype, d_fftsize, 6.76);
     }
+
+    if (!d_window_normalize) {
+        return;
+    }
+    // Calculate power of the window as 1/N * sum of squares
+    const float power =
+        boost::accumulate(
+            d_window | boost::adaptors::transformed([](const float x) { return x * x; }),
+            0.0) /
+        d_window.size();
+    const double norm_factor = std::sqrt(power);
```

Comment:
Is it worth using `1 / sqrt(power)` instead, to save on the divisions?

Reply:
Possibly. This is not high-perf code though. I would have used your suggestion had I seen it earlier -- note how I brainfarted this whole operation.